### ISSUE TITLE
Font ligatures not rendering in VSCode

### ISSUE THREAD

**Issue by absozero** [tonsky/FiraCode#1231]
Even though I set the font ligature option to true, the font does not render the character I want it to. I have meticulously followed the instructions on installing the font I need. It would awesome to have help on why the ligatures are not rendering. I am coding in python. 

![image](https://user-images.githubusercontent.com/77213354/116659995-41fffe00-a947-11eb-8f9d-781d4ca62ee6.png)

The arrows point to where I have written something that should become a ligature but is acting weird and remains as the normal font, and I don't know why this is happening. Any help would be greatly appreciated

**Comment by alexeyten** [comment]
Have you tried other fonts?

**Comment by absozero** [comment]
Thanks for the help here, and thanks for the fast response. I enabled in both of the settings jsons of the User setting and the Workspace setting, and then it worked. Again, thank you. By the way, this is a beautiful font that I love using over the default vs code config and its curvature is awesome, making it look nice and easy to read at the same time. The ligatures are great, too! That is the reason why I chose and will now on be using this font over jet brains mono and the default vs code font.
